src/Test.UnitTests.Sarif.Multitool.Library/Test.UnitTests.Sarif.Multitool.Library.csproj

```diff
@@ -11,6 +11,7 @@
     <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
+    <RootNamespace>Microsoft.CodeAnalysis.Sarif.Multitool</RootNamespace>
```

Comment:
>Multitool [](start = 48, length = 9)

Without this, the root namespace for newly created files defaults to `Microsoft.CodeAnalysis.Test.UnitTests.Multitool.Library`. That caught me when I created RewriteCommandTests.cs.